config/Weather2/Misc.cfg

```diff
@@ -0,0 +1,146 @@
+    I:Cloud_ParticleSpawnDelay=0
+    S:Dimension_List_Clouds=0,-127
+    S:Dimension_List_Storms=0,-127
+    S:Dimension_List_Weather=0,-127
+    S:Dimension_List_WindEffects=0,-127
+    I:Lightning_OddsTo1OfFire=20
+    I:Lightning_lifetimeOfFire=3
+    I:Misc_AutoDataSaveIntervalInTicks=36000
+    B:Misc_ForceVanillaCloudsOff=true
+    B:Misc_proxyRenderOverrideEnabled=true
+    I:Misc_simBoxRadiusCutoff=1024
+    I:Misc_simBoxRadiusSpawn=1024
+    B:Misc_windOn=true
+    D:Particle_Precipitation_effect_rate=1.0
+    B:Particle_RainSnow=true
+    B:Particle_VanillaAndWeatherOnly=false
+    I:Player_Storm_Deadly_OddsTo1=30
+    I:Player_Storm_Deadly_TimeBetweenInTicks=72000
+    I:Player_Storm_Rain_OddsTo1=150
+    I:Server_Storm_Deadly_OddsTo1=30
+    I:Server_Storm_Deadly_TimeBetweenInTicks=72000
+    B:Server_Storm_Deadly_UseGlobalRate=false
+    B:Snow_ExtraPileUp=false
+    I:Snow_MaxBlockBuildupHeight=3
+    B:Snow_PerformSnowfall=false
+    I:Snow_RarityOfBuildup=64
+    B:Snow_SmoothOutPlacement=false
+    I:Storm_AllTypes_TickRateDelay=60
+    I:Storm_Deadly_CollideDistance=128
+    B:Storm_FlyingBlocksHurt=true
+    I:Storm_HailPerTick=10
+    I:Storm_LightningStrikeBaseValueOddsTo1=200
+
+    # Maximum possible stormfronts (or, more corrently, cloud formations) per player. Reducing this will improve network bandwidth.
+    I:Storm_MaxPerPlayerPerLayer=20
+    I:Storm_MaxRadius=300
+    B:Storm_NoRainVisual=false
+
+    # Makes weather boring! or peacefull?
+    B:Storm_NoTornadosOrCyclones=false
+    I:Storm_OddsTo1OfHighWindWaterSpout=150
+    I:Storm_OddsTo1OfLandBasedStorm=-1
+    I:Storm_OddsTo1OfOceanBasedStorm=300
+    I:Storm_OddsTo1OfProgressionBase=15
+    I:Storm_OddsTo1OfProgressionStageMultiplier=3
+    I:Storm_ParticleSpawnDelay=0
+    I:Storm_Rain_WaterBuildUp=150
+    I:Storm_Rain_WaterBuildUpOddsTo1FromNothing=100
+    I:Storm_Rain_WaterBuildUpOddsTo1FromSource=15
+    I:Storm_Rain_WaterBuildUpRate=10
+    I:Storm_Rain_WaterSpendRate=3
+    D:Storm_TemperatureAdjustRate=0.1
+
+    # Use a list of blocks instead of grabbing based on calculated strength of block
+    B:Storm_Tornado_GrabCond_List=false
+    B:Storm_Tornado_GrabCond_List_PartialMatches=false
+
+    # Grab blocks based on how well a diamond axe can mine the block, so mostly wooden blocks
+    B:Storm_Tornado_GrabCond_StrengthGrabbing=true
+    S:Storm_Tornado_GrabList=planks, leaves
+
+    # Treat block grab list as a blacklist instead of whitelist
+    B:Storm_Tornado_GrabListBlacklistMode=false
+
+    # Prevents tearing up of dirt, grass, sand and logs, overrides all other conditions
+    B:Storm_Tornado_RefinedGrabRules=true
+
+    # Accuracy of tornado aimed at player
+    I:Storm_Tornado_aimAtPlayerAngleVariance=5
+
+    # Make tornados initial heading aimed towards closest player
+    B:Storm_Tornado_aimAtPlayerOnSpawn=true
+
+    # Tear up blocks from the ground based on conditions defined
+    B:Storm_Tornado_grabBlocks=true
```

Comment:
Where are the conditions defined for this, I'd be concerned over tornadoes breaking things that should be on the surface from tech mods. Likely this should be false

Reply:
It's whitelisted by default
```

    # Treat block grab list as a blacklist instead of whitelist
    B:Storm_Tornado_GrabListBlacklistMode=false 
```

Reply:
```
     # Grab blocks based on how well a diamond axe can mine the block, so mostly wooden blocks 
    B:Storm_Tornado_GrabCond_StrengthGrabbing=true
    S:Storm_Tornado_GrabList=planks, leaves 
```

Reply:
Leaves dont grow back though so this will probably end up making places look awful right?

Reply:
... there's a mod that makes them regrow: https://www.curseforge.com/minecraft/mc-mods/regrowable-leaves

Reply:
Should I add that mod in this PR, or PR it separately?

Reply:
That might work though the leaves getting thrown about and placed not on a tree may also look bad. Try it with that mod and see if you can test how it acts.

Reply:
Yeah just disable the block grabbing part

Reply:
done